Tests/SwiftGRPCNIOTests/NIOServerTests.swift

```diff
@@ -300,3 +300,89 @@ extension NIOServerTests {
     waitForExpectations(timeout: defaultTimeout)
   }
 }
+
+// Only test Unary and ServerStreaming, as ClientStreaming is not
+// supported in HTTP1.
+extension NIOServerTests {
+  private func gRPCEncodedMessage(_ string: String) -> Data {
+    var data = string.data(using: .utf8)!
+    data.insert(contentsOf: [UInt8]([0x0a, UInt8(data.count)]), at: 0)
```

Comment:
nit: Please explain the `0xa` byte, and the `0x00` ones in the line below.

Reply:
Done